### PR TITLE
Nerfs flashbangs

### PR DESCRIPTION
**Pull request by hornygranny**
magic missile for security isn't good

### CODE REVIEW

**Comment by optimumtact** [comment]
Smells like somebody got ded

**Comment by hornygranny** [comment]
>implying i don't always steal a bowman and sunglasses

**Comment by Firecage** [comment]
:-1:

**Comment by Steelpoint** [comment]
Definition of 'I ded pls nerf'.

This PR may as well remove Flashbangs from the game, considering Fbangs are already obnoxiously loud items.

**Comment by Thunder12345** [comment]
:-1: just remove flashbangs if you want to nerf them this hard. Leaving them in but this weak will just get unsuspecting officers killed when they're targets get up suddenly.

**Comment by kevinz000** [comment]
I ded pls nerf
Security doesn't need more nerfs.

**Comment by optimumtact** [comment]
Just close your pr bro, you're getting wrecked and shreked

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 1]
```diff
@@ -29,23 +29,21 @@
...
-		M.Stun(15)
+		M.Weaken(5) //hella stunned
+		M.Stun(5)
 		M.adjust_eye_damage(8)
 
 	if(M.flash_eyes(affect_silicon = 1))
-		M.Stun(max(10/distance, 3))
-		M.Weaken(max(10/distance, 3))
+		M.Stun(1)
+		M.confused += (max(10/distance, 3))
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: Should add some actual eye damage to this

**Comment by Core0verload** [comment]
:-1: flashbang ruined my tator, flashbang op, nerf it

File: code/game/objects/items/weapons/grenades/flashbang.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: And maybe make the confusion from the deafness part? Though everyone seems mad enough that this probably isnt worth the trouble

**Comment by Incomptinence** [comment]
Flashbang is the last recourse against meat mob. 

Devalue the meat bring back the grinder.

**Comment by GunHog** [comment]
This nerf seems a bit too strong. Security officers can be completely obnoxious with these things, especially on the shuttle. That being said, this PR may make them too weak to use beyond countering borgs at range (same as the flash is now).

File: code/game/objects/items/weapons/grenades/flashbang.dm (hunk 1 above)
Comment (phil235) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: eye damage is already handled in M.flash_eyes()

**Comment by Steelpoint** [comment]
These flashbangs will only stun a borg for one second.

Good luck.

**Comment by SweaterKittens** [comment]
While Flashbangs are incredibly good in the right hands, they're also really easy to predict and avoid if you have any sort of situational awareness. The only time I really see people getting wrecked by flashbangs is when they're in an enclosed space that they can't get out of or they're very new. I really don't think they need a nerf.

**Comment by Robustin** [comment]
Flashbangs are like the cornerstone of making sec tolerable to play. There's way too many situations where sec is expected to take on retarded odds and flashbangs are really the only way to make things manageable. 

If this was merged I would definitely play less sec, no way am I going to want to take on any team antag with a shitty nerfed version. I'm saying this as someone who plays more antag than sec, I still don't want to see sec lose these things.

**Comment by xxalpha** [comment]
:+1:

**Comment by Zilenan91** [comment]
Currently this is just removing the only decent ranged option sec has that's not a taser and replacing it with nothing. :-1:

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 2]
```diff
@@ -29,23 +29,21 @@
...
-		M.Weaken(15) //hella stunned
-		M.Stun(15)
+		M.Weaken(5) //hella stunned
+		M.Stun(5)
 		M.adjust_eye_damage(8)
 
 	if(M.flash_eyes(affect_silicon = 1))
-		M.Stun(max(10/distance, 3))
-		M.Weaken(max(10/distance, 3))
+		M.Stun(1)
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: Stun(1) is not a stun at all. That's just a disarm.

**Comment by CPTANT** [comment]
Sec is going to need some serious improvements in firepower if this is merged.

**Comment by Malkevin** [comment]
Them stunning fully when they go off in-hand/at-feet wasn't nerf enough?

**Comment by Cebasiz** [comment]
this is a joke right

**Comment by TheNightingale** [comment]
Flashbangs aren't even good.

**Comment by PKPenguin321** [comment]
imo the main nerf flashbangs need are so that if you're already blinded/deafened they can't blind/deafen you a second time
this is overkill

**Comment by GunHog** [comment]
> imo the main nerf flashbangs need are so that if you're already blinded/deafened they can't blind/deafen you a second time
this is overkill

This. This right here. Sec players are AWFUL with these things on the shuttle, and THAT is what needs to stop, multiple of these things chained in succession.

**Comment by MrPerson** [comment]
Would rather see them removed than nerfed this hard. With these changes the item basically has no use case whatsoever.

**Comment by Zilenan91** [comment]
What.

Literally the only thing stopping security from being massive loot pinatas is the boxes and boxes of flashbang spam they get. Why even play the job after this if making a spear and being an assistant is 900% more effective than being security. Hell being a spear assistant already is more effective than security because you don't give them any stuns weapons.

**Comment by AdamElTablawy** [comment]
>nerfing the only way to counter tons of people attacking you at once

bad

**Comment by Antimattercarp** [comment]
Security Officer:
>Stun Baton
>Taser
>Chest armor
>Head Armor
>Sec Jumpsuit
>Jackboots
>~~Flashbangs~~
>Handcuffs

Paranoid Bartender:
>Stunprod
>SxS Shotgun
>Chest Armor
>Head Armor
>Cable cuffs

I am certain I missed some minor things for both sides but you likely get the point I am attempting to make.

**Comment by PKPenguin321** [comment]
not really
if anything you've shown just how well equipped our sec officers are

**Comment by Steelpoint** [comment]
But seriously Flashbangs are a easy to avoid item if your attentive, the risk with them is that the longer you hold the Flashbang in your hand to cook it off and give less warning to run away, the higher the risk of it detonating in your hand is.

What with the lag and ping to /tg/station a flashbang detonating in your hand will stun you for 10 or more seconds irrespective of armour or protection.

I've watched countless times in game (and just recently) where flashbangs are ineffective in open combat, they prove best in area's where the target is limited in movement. 

This nerf will just render flashbangs near useless, just remove them then if your that keen.

**Comment by PKPenguin321** [comment]
>But seriously Flashbangs are a easy to avoid

not if the room they're used in is smaller than a screen, IE if you're hiding somewhere or if sec locks you into the brig

**Comment by Steelpoint** [comment]
If you read my post you'll notice I addressed that.

**Comment by Steelpoint** [comment]
Oh come on at least make any nerf reasonable, this is way to much of a nerf.

Hell you could start by not making the fbang detonation cover the entire 14x14 screen.

This just neuters the flashbang to useless levels.

**Comment by CPTANT** [comment]
A small range reduction would actually be a good change, keeps the flashbang useful without the yolo you got stunned because on the other side of the screen a flashbang went off.

**Comment by demonfiren** [comment]
inb4 this gets through anyway

**Comment by Zilenan91** [comment]
If it gets through I'm just gonna advocate people not play security. Go assistant, get your valids there rather than through sec.

**Comment by PKPenguin321** [comment]
>If it gets through I'm just gonna advocate people not play security. Go assistant, get your valids there rather than through sec.

or just don't suck shit at aiming your taser

**Comment by GunHog** [comment]
i ded plz nerf

**Comment by Zilenan91** [comment]
It stuns for like 5 seconds. On a crowd that's nothing.

**Comment by phil235** [comment]
this is too much nerf for my taste :-1:

**Comment by KorPhaeron** [comment]
Near universal dislike and too far of a nerf anyway. I've got better things to burn all goodwill towards me on (like scaling back the AI)

**Pull request closed without merging**